### PR TITLE
Robot Claim UI

### PR DESCRIPTION
**Pull request by META-DREAMER**
implements UI for ROBOT claims + some cleanup / refactoring

### CODE REVIEW

File: packages/shared/config/getEnvValue.ts [hunk 1]
```diff
@@ -2,15 +2,12 @@ export function getEnvValue<T extends string | number>(
...
   }
 
-  if (defaultVal === undefined)
-    throw new Error(`Missing ENV Variable: ${v} and NO DEFAULT AVAILABLE`);
+  if (defaultVal === undefined) throw new Error('Missing ENV Variable');
 
-  console.warn(`Using default ENV variable: ${defaultVal} for value: ${v}`);
+  console.warn(`Using default ENV variable: ${defaultVal}`);
   return defaultVal;
 }
```
Comment (META-DREAMER) [inline packages/shared/config/getEnvValue.ts:13]: @scottrepreneur I had to revert these changes since it wasn't pulling in env variables correctly. probably some weird node thing where its looking specifically for `process.env.foo` to inject the value

Comment (scottrepreneur) [reply packages/shared/config/getEnvValue.ts:13]: hmm is there a way to retrieve the actual name of the ENV. would help this error message quite a lot.

**Review by scottrepreneur — APPROVED** [review]
